services/docker/docker-version.service.js

```diff
@@ -73,17 +76,15 @@ module.exports = class DockerVersion extends BaseJsonService {
     })
   }
 
-  transform({ tag, sort, data, pagedData }) {
+  transform({ tag, sort, data, pagedData, arch = 'amd64' }) {
     let version
 
     if (!tag && sort === 'date') {
       version = data.results[0].name
       if (version !== 'latest') {
         return { version }
       }
-      const imageTag = data.results[0].images.find(
-        i => i.architecture === 'amd64'
-      ) // Digest is the unique field that we utilise to match images
+      const imageTag = data.results[0].images.find(i => i.architecture === arch) // Digest is the unique field that we utilise to match images
```

Comment:
Yeah, this was done by prettier. I think this can be fixed by setting a tab width config in prettier, although that would probably lead to a lot of unrelated changes. If you want, I can open a separate PR for that.